app/routes.py

```diff
@@ -0,0 +1,54 @@
+from fastapi import FastAPI, HTTPException, Query, Request
+from fastapi.responses import HTMLResponse
+from fastapi.staticfiles import StaticFiles
+from app.templates import templates
+from app.scraping import extract_images
+from app.openai_utils import generate_headline
+from openai import OpenAI
+import os
+from dotenv import load_dotenv
+from pydantic import HttpUrl
+import requests
+from bs4 import BeautifulSoup
+
+# Load environment variables
+load_dotenv()
+
+# Initialize OpenAI API key
+openai_api_key = os.getenv("OPENAI_API_KEY")
+
+app = FastAPI()
+
+# Mount the static files directory
+app.mount("/static", StaticFiles(directory="templates/static"), name="static")
+
+@app.get("/", response_class=HTMLResponse)
+def read_root(request: Request):
```

Comment:
Put this in `main.py`

Reply:
Sure, I can move the initialization of the FastAPI app and the mounting of static files to `main.py`. I'll update the PR accordingly.